manifests/m/MicrosoftCorporation/KinectforWindowsSDKv2/0_1409/2.0.1410.19000/MicrosoftCorporation.KinectforWindowsSDKv2.0_1409.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.KinectforWindowsSDK
+PackageVersion: 2.0.1410.19000
+Installers:
+- Architecture: x86
+  InstallerType: burn
+  InstallerUrl: https://download.microsoft.com/download/F/2/D/F2D1012E-3BC6-49C5-B8B3-5ACFF58AF7B8/KinectSDK-v2.0_1409-Setup.exe
+  InstallerSha256: 1D8AC60932F038BE1C67B9D1177BC97421CB805E38256E0FFF9C9AF414A6F78E

```

Comment:
Also, it looks like the Kinect SDK does not support silent install (https://social.msdn.microsoft.com/Forums/sqlserver/en-US/f1cbd02a-ee59-4546-b1e9-557c29177294/silentunattended-install-of-kinect-sdk?forum=kinectsdk), so it can't be added to the repo right now.

Reply:
Ahh i see; thank you for all the help regardless :)